tests/functional/flows/test_list_flows.py

```diff
@@ -1,8 +1,135 @@
+import datetime
 import urllib.parse
+import uuid
+from typing import Any, Dict
 
 import pytest
+from responses import matchers
 
-from globus_sdk._testing import get_last_request, load_response
+from globus_sdk._testing import (
+    get_last_request,
+    load_response,
+    load_response_set,
+    register_response_set,
+)
+
+OWNER_ID = "e061df5a-b7b9-4578-a73b-6d4a4edfd66e"
+
+
+def generate_hello_world_example_flow(n: int) -> Dict[str, Any]:
+    flow_id = str(uuid.UUID(int=n))
+    # base_time = datetime.datetime.fromisoformat("2021-10-18T19:19:35.967289+00:00")
+    base_time = datetime.datetime(
+        2021, 10, 18, 19, 19, 35, 967289, tzinfo=datetime.timezone.utc
+    )
```

Comment:
Is this comment here to suggest a future code swap with a later version of Python? Would this be a candidate for the `if sys.version_info >= (3, x)` that allows pyupgrade to auto-remove this compatibility call at some point in the future?

Reply:
Ooh, yes, I had forgotten about that trick; I'll make that change.

The comment was left there to suggest a swap in the future. I originally pulled the string from an API response (and then wanted to do datetime math on it), and parsed it in this way.